src/extension/background-script/db.ts

```diff
@@ -1,5 +1,4 @@
 import Dexie from "dexie";
-import "fake-indexeddb/auto";
```

Comment:
@escapedcat do you know why this was loaded here? this should only be loaded in the tests, shouldn't it?

Reply:
I assume this was a mistake